src/ape/api/providers.py

```diff
@@ -659,47 +648,18 @@ def get_block(self, block_id: BlockID) -> BlockAPI:
         return self.network.ecosystem.block_class.decode(block_data)  # type: ignore
 
     def get_nonce(self, address: str) -> int:
-        """
-        The number of transactions sent from an address.
-        """
         return self._web3.eth.get_transaction_count(address)  # type: ignore
 
     def get_balance(self, address: str) -> int:
-        """
-        The balance of the account of a given address.
-        """
         return self._web3.eth.get_balance(address)  # type: ignore
 
     def get_code(self, address: str) -> bytes:
-        """
-        Code at a given address.
-        """
         return self._web3.eth.get_code(address)  # type: ignore
 
     def send_call(self, txn: TransactionAPI) -> bytes:
-        """
-        Execute a new message call immediately without creating a
-        transaction on the block chain.
-        """
         return self._web3.eth.call(txn.as_dict())
 
     def get_transaction(self, txn_hash: str, required_confirmations: int = 0) -> ReceiptAPI:
-        """
```

Comment:
Deleting all web3 provider docs because they are just copied from the `ProviderAPI` and the output will use the bae class docs if we don't set any, so it is less we have to manager by just deleting these.